entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/entity/FlusherBasedEntityLoader.java

```diff
@@ -55,8 +62,10 @@ private String getQueryString() {
                 flushers[i].appendFetchJoinQueryFragment("e", sb);
             }
         }
-        sb.append(" WHERE e.").append(idAttributeName).append(" = :id");
-
+        for (String idAttributeName : idAttributeNames){
+            sb.append(" WHERE e.").append(idAttributeName).append(" = :"+idAttributeName.replace(".","_")+"_variable"+" AND");
```

Comment:
`idAttributeName.replace(".","_")+"_variable"` works because it's used below in the similar fashion. Please extract this to a method `getParameterName(String idAttributeName)`, so we have more safety the two remain using the same naming scheme.